tokio/src/net/windows/named_pipe.rs

```diff
@@ -738,10 +738,48 @@ impl NamedPipeOptions {
     /// be specified for other instances of the pipe. Acceptable values are in
     /// the range 1 through 254. The default value is unlimited.
     ///
+    /// This value is only respected by the first instance that creates the pipe
+    /// and is ignored otherwise.
+    ///
     /// This corresponds to specifying [`nMaxInstances`].
     ///
     /// [`nMaxInstances`]: https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createnamedpipea
     ///
+    /// # Errors
+    ///
+    /// The same numbers of `max_instances` have to be used by all servers. Any
+    /// additional servers trying to be built which uses a mismatching value
+    /// will error.
+    ///
+    /// ```
+    /// use std::io;
+    /// use tokio::net::windows::{NamedPipeOptions, NamedPipeClientOptions};
+    ///
+    /// // Use this from winapi instead in real code.
+    /// const ERROR_PIPE_BUSY: u32 = 231;
```

Comment:
I'm pretty sure we can use winapi in doc-tests?

Reply:
yeah, fixed